src/situatedSIPP/learningAlgorithms/DijkstraLearning.cpp

```diff
@@ -41,13 +40,12 @@ void DijkstraLearning::learn(OPEN_container& open, std::unordered_multimap<int,
         // reference code https://github.com/gtianyi/rationalRealtimeSearch/blob/master/cpp/learningAlgorithms/Dijkstra.h
         // keep track of heuristic values somewhere.
         std::unordered_set<Node>::iterator cit;
-        double acc = NAN;
-        Node * n = nullptr;
+        const Node * n = nullptr;
```

Comment:
We should avoid using meaningless variable names. It's very hard for teammates to understand the code.